tests/cli/test_status.py

```diff
@@ -31,8 +31,8 @@
-ROUTE6 = '[{"type":"unicast","dst":"::1","dev":"lo","protocol":"kernel","scope":"global","metric":256,"flags":[],"pref":"medium"},{"type":"unicast","dst":"2001:9e8:a19f:1c00::/64","dev":"enp0s31f6","protocol":"ra","scope":"global","metric":100,"flags":[],"expires":7199,"pref":"medium"},{"type":"unicast","dst":"2001:9e8:a19f:1c00::/64","dev":"wlan0","protocol":"ra","scope":"global","metric":600,"flags":[],"pref":"medium"},{"type":"unicast","dst":"2001:9e8:a19f:1c00::/56","gateway":"fe80::cece:1eff:fe3d:c737","dev":"enp0s31f6","protocol":"ra","scope":"global","metric":100,"flags":[],"expires":1799,"pref":"medium"},{"type":"unicast","dst":"2001:9e8:a19f:1c00::/56","gateway":"fe80::cece:1eff:fe3d:c737","dev":"wlan0","protocol":"ra","scope":"global","metric":600,"flags":[],"pref":"medium"},{"type":"unicast","dst":"2001:dead:beef::/64","dev":"tun0","protocol":"kernel","scope":"global","metric":256,"flags":[],"pref":"medium"},{"type":"unicast","dst":"fe80::/64","dev":"enp0s31f6","protocol":"kernel","scope":"global","metric":256,"flags":[],"pref":"medium"},{"type":"unicast","dst":"fe80::/64","dev":"wlan0","protocol":"kernel","scope":"global","metric":1024,"flags":[],"pref":"medium"},{"type":"unicast","dst":"default","gateway":"fe80::cece:1eff:fe3d:c737","dev":"enp0s31f6","protocol":"ra","scope":"global","metric":100,"flags":[],"expires":1799,"metrics":[{"mtu":1492}],"pref":"medium"},{"type":"unicast","dst":"default","gateway":"fe80::cece:1eff:fe3d:c737","dev":"wlan0","protocol":"ra","scope":"global","metric":20600,"flags":[],"pref":"medium"}]'  # nopep8
+ROUTE4 = '[{"type":"unicast","dst":"default","gateway":"192.168.178.1","dev":"enp0s31f6","table":"main","protocol":"dhcp","scope":"global","prefsrc":"192.168.178.62","metric":100,"flags":[]},{"type":"unicast","dst":"default","gateway":"192.168.178.1","dev":"wlan0","table":"main","protocol":"dhcp","scope":"global","metric":600,"flags":[]},{"type":"unicast","dst":"10.10.0.0/24","dev":"wg0","table":"main","protocol":"kernel","scope":"link","prefsrc":"10.10.0.2","flags":[]},{"type":"unicast","dst":"192.168.178.0/24","dev":"enp0s31f6","table":"main","protocol":"kernel","scope":"link","prefsrc":"192.168.178.62","metric":100,"flags":[]},{"type":"unicast","dst":"192.168.178.0/24","dev":"wlan0","table":"main","protocol":"kernel","scope":"link","prefsrc":"192.168.178.142","metric":600,"flags":[]},{"type":"unicast","dst":"192.168.178.1","dev":"enp0s31f6","table":"main","protocol":"dhcp","scope":"link","prefsrc":"192.168.178.62","metric":100,"flags":[]},{"type":"broadcast","dst":"192.168.178.255","dev":"enp0s31f6","table":"local","protocol":"kernel","scope":"link","prefsrc":"192.168.178.62","flags":[]}]'  # nopep8
```

Comment:
thought: What about numeric routing tables, which do not have a symbolic name in `/etc/iproute2/rt_table`? Can we have a test case for those, too? We seem to be only testing for "local" and "main" here.

Reply:
I changed one of them to a number. I wasn't expecting any differences because it is represented as a string in the json even if it's a number.